shapely/ctypes_declarations.py

```diff
@@ -288,6 +288,9 @@ def prototype(lgeos, geos_version):
     lgeos.GEOSCovers.restype = c_byte
     lgeos.GEOSCovers.argtypes = [c_void_p, c_void_p]
 
+    lgeos.GEOSCoveredBy.restype = c_bool
```

Comment:
I suppose (but didn't try) this can be `c_byte`, since all the other predicates here are using that as well?

Reply:
You're right, we didn't need `c_bool`. I've reverted to `c_byte`.